_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -27,6 +27,11 @@
 respondents = int(os.getenv('test_respondents'))
 logger = logging.getLogger()
 
+requests_filepath = os.getenv('json_requests_filepath', 'requests.json')
```

Comment:
There's probably a better way of doing this but usually this would be in a method to `get_requests_list` or something and invoked at the point it's required. However as this is a script we'll leave it as is. If the file is missing it errors right at the start (rather than prepping all the data so there's one benefit as is).

Reply:
For some reason, if I put a logging statement in this block of code, it doesn't show up in the logs unless the level is set to `error` instead of `info`, unlike the other logging statements.

Reply:
The filename is good enough for the time being as we'll keep them all be in the same folder

Reply:
Done. The logging now shows up when set to `info`.

Reply:
Logging working as expected at info level